internal/kafka/test/integration/kafka_create_test.go

```diff
@@ -0,0 +1,528 @@
+package integration
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/constants"
+	constants2 "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/constants"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/public"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/kafkas/types"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/presenters"
+	kafkatest "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test/common"
+	mockkafkas "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test/mocks/kafkas"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test/mocks/kasfleetshardsync"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/metrics"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/test/mocks"
+
+	"github.com/onsi/gomega"
+)
+
+// Test Kafka creation with manual scaling enabled
+func TestKafkaCreate_ManualScaling(t *testing.T) {
+	g := gomega.NewWithT(t)
+
+	ocmServer := mocks.NewMockConfigurableServerBuilder().Build()
+	defer ocmServer.Close()
+
+	var enableManualScaling bool
+	clusterList := config.ClusterList{}
+	h, client, teardown := kafkatest.NewKafkaHelperWithHooks(t, ocmServer, func(d *config.DataplaneClusterConfig) {
+		if enableManualScaling {
+			// Enable manual scaling and register data plane cluster to cluster config
+			d.DataPlaneClusterScalingType = config.ManualScaling
+			d.ClusterConfig = config.NewClusterConfig(clusterList)
+		}
+	})
+	defer teardown()
+
+	// run mock fleetshard sync
+	mockKasFleetshardSyncBuilder := kasfleetshardsync.NewMockKasFleetshardSyncBuilder(h, t)
+	mockKasFleetshardSync := mockKasFleetshardSyncBuilder.Build()
+	mockKasFleetshardSync.Start()
+	defer mockKasFleetshardSync.Stop()
+
+	// set up data plane cluster
+	clusterID, getClusterErr := common.GetOSDClusterIDAndWaitForStatus(h, t, api.ClusterReady)
+	g.Expect(getClusterErr).ToNot(gomega.HaveOccurred(), "failed to get dataplane cluster details: %v", getClusterErr)
+	g.Expect(clusterID).ToNot(gomega.BeEmpty(), "no dataplane cluster found")
```

Comment:
This might not be needed as we can right away use the manual list to put the cluster in the database in a given status.
This can save us some test time

Reply:
i.e just keep what's done in line 68 with some hardcoded values. 

Same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1167/files#r924748354

Reply:
Already addressed